createdisk.sh

```diff
@@ -344,6 +344,9 @@ ${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/rhcos-${ostree_hash}/
 # Get the IP of the VM
 internalIP=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'hostname -i')
 
+# Add a dummy network interface with internalIP
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo nmcli conn add type dummy ifname eth10 con-name dummy ip4 ${internalIP}/24  && sudo nmcli conn up dummy"
```

Comment:
we can use a more descriptive name than `dummy`. Maybe `internalEtcd` since this is what it's going to be used for?

Reply:
I believe a much more detailed commit log would be useful here

>  Add a dummy network using internal IP

> As part of etcd the certs are created for internalIP instead of srv records. This network IP is going to be instance IP so that the certs are always valid for it even running on different platform.

The problem we are trying to solve is that the cluster/node IP is going to change depending on the platform CRC runs on, and the etcd certificate is IP only. How we are solving it, by adding a dummy network interface on the node whose IP will match the one expected by etcd, and we'll make sure the cluster uses this internally. All of this is worthwhile to have in the log in my opinion.